### ISSUE TITLE
Allow Year/Semester and level selection from the interface

### ISSUE THREAD

**Issue by Shastick** [Shastick/orgarec#12]

**Comment by ChrisJamesC** [comment]
Do you think that it would be better to load the whole graph from the database in the snippet once and to then filter in scala? 
To me, even if the current implementation doesn't allow it, we should re-load the whole graph from the database when changing these parameters, then compute a diff, and then update points.

**Comment by Shastick** [comment]
Yes, for new year/level/section parameters a new graph must be created.

It can then be stored in the LandscapeHolder session variable (it's 
currently the case)

there's no point in computing a diff at the moment.

by the way: the way it's done now, two people watching the same graph 
will experience weird things as StudyPlanCompleteGraph is a single 
object on the server that is accessed by everybody ;-)

**Comment by ChrisJamesC** [comment]
The code is now corrected for the service to be usable by multiple users.
